polar-core/Cargo.toml

```diff
@@ -32,7 +32,6 @@ permute = "0.1.0"
 pipe = "0.4.0"
 pretty_assertions = "1.0.0"
 maplit = "1.0.2"
-indoc = "1.0.3"
```

Comment:
Is this right? It looks like you're still using this?

Reply:
This is removing it from dev deps — it's still in regular deps

Reply:
here: https://github.com/osohq/oso/pull/1393/files/ecd33230343e4a34684dc6151f303e8a83757175#diff-21e05eb50ab6604a97135d0c32a948d7e8408b5997f8fcb818029f8cf716c215L23